README.md

```diff
@@ -209,6 +209,21 @@ If you need to override some of the global settings latter for a specific env yo
 ```
 composer require fourkitchens/fire --dev --prefer-install=source
 ```
+
+## Passing Arguments to Lando/DDEV
+With FIRE, you can pass unknown arguments to Lando/DDEV commands using a double dash `--`. This is useful when you need to run Drush commands with additional options. Here's how to do it:
```

Comment:
I'm not sure about this description and title,  I was thinking something more along these lines:


## Passing Arguments wrapped commands (i.e drush)

If you are using fire to run a wrapped command like drush and you need to pass arguments to your drush command you just need to use a double hyphen and then specify your hyphenated arguments, example:

`fire drush cex -- -y`



Feel free to add or remove details @Yuvania

Reply:
Resolved